flink-java/src/test/java/org/apache/flink/api/java/utils/ParameterToolTest.java

```diff
@@ -454,124 +469,196 @@ public void testUnrequestedDoubleWithMissingValue() {
-		Assert.assertEquals("∞", parameter.getRequired("required"));
-		Assert.assertEquals(Collections.emptySet(), parameter.getUnrequestedParameters());
+		assertEquals("∞", parameter.getRequired("required"));
+		assertEquals(Collections.emptySet(), parameter.getUnrequestedParameters());
 
 		// test repeated access
-		Assert.assertEquals("∞", parameter.getRequired("required"));
-		Assert.assertEquals(Collections.emptySet(), parameter.getUnrequestedParameters());
+		assertEquals("∞", parameter.getRequired("required"));
+		assertEquals(Collections.emptySet(), parameter.getUnrequestedParameters());
 	}
 
 	@Test
 	public void testUnrequestedMultiple() {
 		ParameterTool parameter = ParameterTool.fromArgs(new String[]{"-boolean", "true", "-byte", "1",
 			"-short", "2", "-int", "4", "-long", "8", "-float", "4.0", "-double", "8.0", "-string", "∞"});
-		Assert.assertEquals(createHashSet("boolean", "byte", "short", "int", "long", "float", "double", "string"),
+		assertEquals(createHashSet("boolean", "byte", "short", "int", "long", "float", "double", "string"),
 			parameter.getUnrequestedParameters());
 
 		Assert.assertTrue(parameter.getBoolean("boolean"));
-		Assert.assertEquals(createHashSet("byte", "short", "int", "long", "float", "double", "string"),
+		assertEquals(createHashSet("byte", "short", "int", "long", "float", "double", "string"),
 			parameter.getUnrequestedParameters());
 
-		Assert.assertEquals(1, parameter.getByte("byte"));
-		Assert.assertEquals(createHashSet("short", "int", "long", "float", "double", "string"),
+		assertEquals(1, parameter.getByte("byte"));
+		assertEquals(createHashSet("short", "int", "long", "float", "double", "string"),
 			parameter.getUnrequestedParameters());
 
-		Assert.assertEquals(2, parameter.getShort("short"));
-		Assert.assertEquals(createHashSet("int", "long", "float", "double", "string"),
+		assertEquals(2, parameter.getShort("short"));
+		assertEquals(createHashSet("int", "long", "float", "double", "string"),
 			parameter.getUnrequestedParameters());
 
-		Assert.assertEquals(4, parameter.getInt("int"));
-		Assert.assertEquals(createHashSet("long", "float", "double", "string"),
+		assertEquals(4, parameter.getInt("int"));
+		assertEquals(createHashSet("long", "float", "double", "string"),
 			parameter.getUnrequestedParameters());
 
-		Assert.assertEquals(8, parameter.getLong("long"));
-		Assert.assertEquals(createHashSet("float", "double", "string"),
+		assertEquals(8, parameter.getLong("long"));
+		assertEquals(createHashSet("float", "double", "string"),
 			parameter.getUnrequestedParameters());
 
-		Assert.assertEquals(4.0, parameter.getFloat("float"), 0.00001);
-		Assert.assertEquals(createHashSet("double", "string"),
+		assertEquals(4.0, parameter.getFloat("float"), 0.00001);
+		assertEquals(createHashSet("double", "string"),
 			parameter.getUnrequestedParameters());
 
-		Assert.assertEquals(8.0, parameter.getDouble("double"), 0.00001);
-		Assert.assertEquals(createHashSet("string"),
```

Comment:
unnecessary changes.

Reply:
True, I guess that was done due to auto reformatting when writing the `testConcurrentExecutionConfigSerialization`.